### PR TITLE
new EraStakers interface support

### PR DESCRIPTION
**Pull request by vovunku**

### CODE REVIEW

**Comment by vovunku** [comment]
Screenshot from 2023-12-28 08-56-44
Screenshot from 2023-12-28 09-02-27
Screenshot from 2023-12-28 09-02-19
Screenshot from 2023-12-27 10-46-09

File: src/mappings/era/ValidatorEraInfoDataSource.ts [hunk 1]
```diff
@@ -43,4 +52,37 @@ export class ValidatorEraInfoDataSource extends CachingEraInfoDataSource {
...
+            const others = exposure.others.map(({who, value}) => {
+                return {
+                    address: who.toString(),
+                    amount: value.toBigInt()
+                }
+            });
+    
+            (accumulator[validatorAddress] = accumulator[validatorAddress] || []).push(...others);
+            return accumulator;
+        }, {})
```
Comment (valentunn) [inline src/mappings/era/ValidatorEraInfoDataSource.ts:73]: This does not guarantee that pages will be ordered. I better suggest to use approach similar to what we use in the client code

image

**Review by valentunn — CHANGES_REQUESTED** [review]

**Comment by vovunku** [comment]
Screenshot from 2023-12-28 11-40-17

**Review by valentunn — APPROVED** [review]

**Review by ERussel — APPROVED** [review]

**Pull request merged**